### ISSUE TITLE
Changing the directory name YouPHPTube

### ISSUE THREAD

**Issue by md97** [DanielnetoDotCom/YouPHPTube#912]
Amazing script. 
Pretty amateur issue... 
How can I get the YouPHPTube script to be on the url of just my domain?

Right now I have to go mydomain.com/YouPHPTube... so how can I get YouPHPTube at simply mydomain.com? When I try move the contents of YouPHPTube directory or change the name to something else it doesn't work. 

I'm using AWS EC2, steps followed in the video tutorials. Any help is greatly appreciated!

**Comment by Nisarg22** [comment]
Move all the folders and files from youphptube folder and delete youphptube

**Comment by DanielnetoDotCom** [comment]
yes, it may work, but I prefer the apache virtual host solution, just google for that

**Comment by honeyfairy** [comment]
you mean this:

`# Ensure that Apache listens on port 80
Listen 80
<VirtualHost *:80>
    DocumentRoot "/www/example1"
    ServerName www.example.com

    # Other directives here
</VirtualHost>

<VirtualHost *:80>
    DocumentRoot "/www/example2"
    ServerName www.example.org

    # Other directives here
</VirtualHost>`

**Comment by DanielnetoDotCom** [comment]
yes, something like that

**Comment by timmayy** [comment]
so all I did was:
1. sudo nano var/www/html/YouPHPTube/videos/configuration.php

2. change the url(s) names to whatever you want.
$global['webSiteRootURL'] = 'https://example.com/**YouPHPTube**/';
$global['systemRootPath'] = '/var/www/html/**YouPHPTube**/';

**Change to;**
$global['webSiteRootURL'] = 'https://example.com/**NewName**/';
$global['systemRootPath'] = '/var/www/html/**NewName**/';

3. **Then rename the folder** itself to reflect what you just changed.

**Comment by md97** [comment]
@timmayy 
So I did what you describe here but now my page is almost a blank html page...
it loads text and links but no styles/buttons/colors...
Any suggestions?

**Comment by timmayy** [comment]
ya i forgot it took me 6 hours to realize i forgot the / haha

glad it worked.
